core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -11,19 +11,28 @@
 
 package alluxio.master.block;
 
+import static alluxio.stress.rpc.TierAlias.MEM;
```

Comment:
If you just use this, you can define `TierAlias` to a Class in the common module.